project/Dependencies.scala

```diff
@@ -39,16 +44,19 @@ object Dependencies {
     "com.typesafe.akka"             %% "akka-slf4j"                % akkaV,
     "com.typesafe.akka"             %% "akka-stream"               % akkaV,
 
-    "org.elasticsearch.client"       % "transport"           % "5.4.3",
-
-    "com.google.guava"               % "guava"               % "28.1-android",
+    "org.elasticsearch.client"       % "transport"           % "5.6.16"
+      exclude("io.netty", "netty-codec")
+      exclude("io.netty", "netty-transport")
+      exclude("io.netty", "netty-resolver")
+      exclude("io.netty", "netty-buffer")
+      exclude("io.netty", "netty-common"),
```

Comment:
I tried an `excludeAll(ExclusionRule(organization = "io.netty"))`, but that caused problems at runtime - there's some netty library in there we DON'T want to exclude.